### PR TITLE
INTL

### PR DESCRIPTION
**Pull request by zakrzaq**
- [x] i18n configure
- [x] persisent lang selection
- [x] pl, en langs
- [x] other minor fixes"

### CODE REVIEW

File: src/components/header/LocalePicker.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  return ["en", "pl"].map((locale) => {
+    return {
+      id: locale,
+      name: locale.toUpperCase(),
+    };
+  });
+});
+
+const handleLocaleChange = () => {
+  localStorage.setItem("LOCALE", t.locale.value);
```
Comment (szymonlicau) [inline src/components/header/LocalePicker.vue:16]: Considering this string name is re-used in multiple places, maybe we could have it refactored to a constant?
Or perhaps even getting/updating this data (the current locale) moved to a composable instead? 👀
I also seen you've put it in a pinia store, we could also use that, and have actions to update it (and save in local storage as a side effect) 😄

File: src/views/ReservationSuccessPage.vue [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <h1 class="success__title">
+      Hell yeah! <br />
+      <span class="success__title--red">You booked {{ 3 }} tickets</span>
+    </h1>
+    <div class="sucess__tickets">
+      <!-- <ReservationItem -->
+      <!--   v-for="ticket in bookedTickets" -->
+      <!--   :key="ticket.seat" -->
+      <!--   :ticket="ticket" -->
+      <!-- /> -->
```
Comment (szymonlicau) [inline src/views/ReservationSuccessPage.vue:44]: Lots of comments in this file, what's the reason for that? 🤔

**Review by szymonlicau — CHANGES_REQUESTED** [review]

File: src/views/ReservationSuccessPage.vue (hunk 2 above)
Comment (zakrzaq) [reply src/views/ReservationSuccessPage.vue:44]: It's a component that is work in progress. Technical debt - I am wanting to finish it on the current branch that i am working as there will be elements overlapping

Comment (szymonlicau) [reply src/views/ReservationSuccessPage.vue:44]: Got it! 👌

File: src/components/header/LocalePicker.vue (hunk 1 above)
Comment (zakrzaq) [reply src/components/header/LocalePicker.vue:16]: I shall refactor this. I should have cleaned up better. I was planning to use pinia but give there is not really that much code there i stuck with localStorage

Comment (zakrzaq) [reply src/components/header/LocalePicker.vue:16]: In truth I am having bit of trouble with composables, I was hoping to ask you about at daily :(

**Review by szymonlicau — APPROVED** [review]

**Pull request merged**